packages/overdrive/lib/components/private/InputBase/NotchedBase.tsx

```diff
@@ -75,15 +92,21 @@ export const NotchedBase: FunctionComponent<Props> = ({
 							borderColourClassName,
 						)}
 						style={{ width: isEmpty ? 0 : notchedWidth }}>
-						<label
+						<Box
```

Comment:
This guy's colour is off, read comments above about select perhaps??

Reply:
Not sure that is the case not after last changes anyway. Same colour values with master
<img width="1104" alt="Untitled 3" src="https://user-images.githubusercontent.com/1334204/88344111-42e77c00-cd86-11ea-9a46-4706f4505635.png">

Reply:
Yeah looks like it.